msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -824,6 +824,7 @@ Specify meeting extension definition. For more information, see [custom Together
 |---|---|---|---|---|
 |`scenes`|array of objects| 5 items||Meeting supported scenes.|
 |`supportsStreaming`|Boolean|||A value that indicates whether an app can stream the meeting's audio and video content to a real-time meeting protocol (RTMP) endpoint. The default value is **false**.|
+|`supportsAnonymousGuestUsers`|Boolean|WIP|WIP|WIP|
```

Comment:
What's "WIP"

Reply:
We are awaiting on the updates/ Inputs for this section, and we have added place holders as WIP.